src/widget.cpp

```diff
@@ -1399,7 +1443,8 @@ std::string widget::layout( const avatar &ava, const unsigned int max_width, int
             // Subtract column padding to get space available for widgets
             const int avail_width = max_width - col_padding * ( num_widgets - 1 );
             // Divide available width equally among all widgets
-            const int child_width = avail_width / num_widgets;
+            // TODO: This - 1 is making me nervous. It's here because it makes sidebars behave better, but I don't know why it's necessary. It shouldn't be necessary, and it scares me.
+            const int child_width = ( avail_width / num_widgets ) - 1;
```

Comment:
For some reason, subtracting 1 from the width of each child widget is making the whole sidebar behave much better. I don't know why, and it scares me.